app/javascript/src/utils/compiler.js

```diff
@@ -282,15 +283,54 @@ function evaluateConditionals(joinedItems) {
     }
 
     const openingBracketIndex = startBracketMatch.index + startBracketMatch[0].length - 1
-    const closingBracketIndex = getClosingBracket(joinedItems, "{", "}", openingBracketIndex - 1)
+    let closingBracketIndex = getClosingBracket(joinedItems, "{", "}", openingBracketIndex - 1)
     if (closingBracketIndex < 0) {
       continue
     }
 
     let conditionalEndingIndex = closingBracketIndex
 
-    const trueBlockContent = joinedItems.substring(openingBracketIndex + 1, closingBracketIndex)
+    let trueBlockContent = joinedItems.substring(openingBracketIndex + 1, closingBracketIndex)
     let falseBlockContent = ""
+    let passed = evaluateExpressionTree(conditionExpressionTree)
+
+    elifStartRegex.lastIndex = closingBracketIndex
+    let elifMatch = elifStartRegex.exec(joinedItems)
+
+    while (elifMatch != null && elifMatch.index === closingBracketIndex + 1) {
+      const openingElifConditionParenIndex = elifMatch.index + elifMatch[0].length - 1
+      const closingElifConditionParenIndex = getClosingBracket(joinedItems, "(", ")", openingElifConditionParenIndex - 1)
+      if (closingElifConditionParenIndex < 0) {
+        break
+      }
+
+      const conditionElifExpression = joinedItems.substring(openingElifConditionParenIndex + 1, closingElifConditionParenIndex)
+      const conditionElifExpressionTree = getExpressionTree(conditionElifExpression)
+
+      startBracketRegex.lastIndex = closingElifConditionParenIndex + 1 // set start position for the exec below
+      const startElifBracketMatch = startBracketRegex.exec(joinedItems)
+      if (startElifBracketMatch == null || startElifBracketMatch.index !== closingElifConditionParenIndex + 1) {
+        break
+      }
+
+      const openingElifBracketIndex = startElifBracketMatch.index + startElifBracketMatch[0].length - 1
+      closingBracketIndex = getClosingBracket(joinedItems, "{", "}", openingElifBracketIndex - 1)
+      if (closingBracketIndex < 0) {
+        break
+      }
+
+      conditionalEndingIndex = closingBracketIndex
+
+      if(!passed) {
```

Comment:
Missing space after if 
```suggestion
      if (!passed) {
```